examples/meta/src/kernel/linear.sg

```diff
@@ -0,0 +1,21 @@
+File traindat = '../data/fm_train_real.dat'
```

Comment:
in all the available examples these lines are different. see for example:
https://github.com/shogun-toolbox/shogun/blob/develop/examples/meta/src/binary/domainadaptationsvm.sg

but any examples currently in the repo...

Reply:
I'm sorry, I fixed that. Now I'm stuck at this part 

```Normalizer avg_diag_normalizer = normalizer("AvgDiagKernelNormalize")``` 

make meta_example throws this error 

```translate.TranslationFailure: 'Failed to obtain include path for Cnormalizer or normalizer or Cnormalizer or normalizer' ```.

Reply:
yes unfortunately kernel normalizers are not yet part of the base interface. there's a PR for this https://github.com/shogun-toolbox/shogun/pull/4671 that should be rebased and merged. until then you can just create the class itself: 
```
AvgDiagKernelNormalize kernel_normalizer = AvgDiagKernelNormalize()
```

Reply:
It still throws the same error. I presume that once that PR is merged, make meta_examples shouldn't throw that error, right?

Reply:
i'm not so sure what you mean... the very same error it should not throw... that pr needs to be fixed.

Reply:
not you should just simply update this PR with your latest changes to be able to discuss further... plz remove the parts i've mentioned above

Reply:
SImilar error I meant, 

```translate.TranslationFailure: 'Failed to obtain include path for CAvgDiagKernelNormalize or AvgDiagKernelNormalize or CAvgDiagKernelNormalize or AvgDiagKernelNormalize'```

Reply:
plz check the class name....